### PR TITLE
[stable/gocd] Changed kubernetes plugin version to v1.0.0

### PR DESCRIPTION
**Pull request by bdpiprava**

### CODE REVIEW

File: stable/gocd/templates/NOTES.txt [hunk 1]
```diff
@@ -1,52 +1,37 @@
...
-  http://{{ . }}
+    http://{{ . }}
 {{- end }}
 {{- else if contains "NodePort" .Values.server.service.type }}
-    export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "gocd.fullname" . }}-server)
-    export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
-    echo http://$NODE_IP:$NODE_PORT
+    $ export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "gocd.fullname" . }}-server)
+    $ export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+    $ echo http://$NODE_IP:$NODE_PORT
```
Comment (unguiculus) [inline stable/gocd/templates/NOTES.txt]: Putting a `$` in front breaks copy/pasting multiple lines.

**Comment by unguiculus** [comment]
cc @arvindsv @ganeshspatil @varshavaradarajan

File: stable/gocd/templates/NOTES.txt (hunk 1 above)
Comment (bdpiprava) [reply stable/gocd/templates/NOTES.txt]: Done. Removed `$` from all the lines.

**Pull request merged**